app/src/main/java/com/example/bigowlapp/utils/SmsSender.java

```diff
@@ -0,0 +1,22 @@
+package com.example.bigowlapp.utils;
+
+import android.content.Context;
+import android.telephony.SmsManager;
+import android.widget.Toast;
+
+public class SmsSender {
+
+
+    private static final String SMS_RECEIVED = "android.provider.Telephony.SMS_RECEIVED";
+
+    public SmsSender() {
+
+    }
+
+    public void sendSMS(Context context, String smsNumber, String smsMessage) {
```

Comment:
you could probably make this static, in which case you should me the constructor private.

Reply:
Yes, I can. I agree with that